hypergan/discriminators/pyramid_discriminator.py

```diff
@@ -76,11 +78,13 @@ def should_ignore_layer(layer, resize):
     if(config['noise']):
         net += tf.random_normal(net.get_shape(), mean=0, stddev=config['noise'], dtype=gan.config.dtype)
         
+    filterw=3
+    filterh=1
```

Comment:
Whether or not this works, I like the idea of supporting 1-d conv.  Perhaps we should move filter to a hyperparam.